src/ResourceManager/Automation/Commands.Automation/Cmdlet/NewAzureAutomationModule.cs

```diff
@@ -39,6 +39,7 @@ public class NewAzureAutomationModule : AzureAutomationBaseCmdlet
         /// </summary>
         [Parameter(Position = 3, Mandatory = true, ValueFromPipelineByPropertyName = true,
             HelpMessage = "The url to a module zip package.")]
+        [Alias("ContentLinkUri")]
```

Comment:
Does changing he parameter name will be breaking change, i.e., does user already using the cmdlet need to do any thing? If this is not the case, fine to do as suggested.